### ISSUE TITLE
IE + JAWS 2019, 2020 - tabindex on container impacts legend announcement

### ISSUE THREAD

**Issue by wardav** [FreedomScientific/VFO-standards-support#480]
### Summary

When a standard fieldset and legend is contained in a `<div>` that has `tabindex`, it appears to prompt JAWS to not announce the legend. This may be similar to https://github.com/FreedomScientific/VFO-standards-support/issues/106.

The tabindex attribute does not need to be a direct parent. I came across this when testing a page that had `tabindex="-1"` on a div for the main region, which then had the radio group as a distant ancestor. 

The issue appears to happen across various of JAWS (including 2019 and 2020). It appears to only happen with IE, though. It appears to work with Firefox and Chrome (with combinations of JAWS and NVDA). So I assume this is IE specific. 

### Example

```
<div tabindex="-1">
  <h2>Nested in div with tabindex</h2>
  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
  <fieldset>
  <legend>Choose a shipping method 1</legend>
    <input id="overnight" type="radio" name="shipping" value="overnight">
    <label for="overnight">Overnight</label><be>
    <input id="twoday" type="radio" name="shipping" value="twoday">
    <label for="twoday">Two day</label><br>
</fieldset>
</div>
```

**Comment by willslone** [comment]
@wardav was unable to reproduce in IE11 with JAWS 2020 and 2021 using your provided code snippet.

**Comment by JAWS-test** [comment]
The problem occurs up to version 2019 and not from version 2020
